lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
I don't know Underscore and JS rules in detail - that makes more sense.
Perhaps Ned can weigh in when he arrives and we can update the
documentation appropriately.

On Mon, Aug 31, 2015 at 10:13 AM, Christina Roberts <
notifications@github.com> wrote:

> In
> lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js
> <https://github.com/edx/edx-platform/pull/9513#discussion_r38315402>:
>
> > @@ -9,7 +9,7 @@
> >          function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
> >              return Backbone.View.extend({
> >
> > -                errorMessage: gettext("An error occurred. Try again."),
> > +                errorMessage: _.escape(gettext("An error occurred. Try again.")),
>
> That seems reasonable, but then the change should be to make the templates
> do the escaping, not to escape every string sent to the templates.
>
> @muhammad-ammar <https://github.com/muhammad-ammar> Can you update the
> templates instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/9513/files#r38315402>.
>